.travis.yml

```diff
@@ -12,6 +12,9 @@ addons:
       - libgles2-mesa-dev
 script:
   - (cd webrender_traits && cargo test --verbose)
+  - (cd webrender_traits && cargo test --verbose --features "ipc")
```

Comment:
Please put this line the first one in the script to avoid unnecessary recompiles.